intera_interface/src/intera_interface/limb.py

```diff
@@ -146,10 +174,14 @@ def __init__(self, limb="right", synchronous_pub=False):
                    "from %s") % (self.name.capitalize(), ns + 'endpoint_state')
         intera_dataflow.wait_for(lambda: len(self._cartesian_pose.keys()) > 0,
```

Comment:
Endpoint_state only gives the 'right_hand' tip. Tip_states gives 'right_hand' and all other tips.

Endpoint_state is redundant now, but I didn't want to change the API to it.